makefile

```diff
@@ -16,8 +16,14 @@ build-demo-react-admin:
 
 build: build-ra-auth-msal build-demo-react-admin ## compile ES6 files to JS
 
-start: ## Start the demo
+start-demo: ## Start the demo
 	@cd ./packages/demo-react-admin && yarn start
 
+start-fake-api: ## Start the fake API
+	@cd ./packages/demo-fake-api && yarn start
+
+start: ## Start the demo
+	@(trap 'kill 0' INT; ${MAKE} start-fake-api & ${MAKE} start-demo)
```

Comment:
For my info, what is this voodoo?

Reply:
I wanted a way to run both the fake-api and the demo in a single comment, but also kill both processes with a CTRL-C.
Maybe there's an easier way to do it, but I found this one here: https://stackoverflow.com/a/52033580/3975522 :grin: 
What do you think? Can we keep it?

Reply:
That's nice!